src/components/FormType.tsx

```diff
@@ -38,11 +39,13 @@ const CalendarFormType = () => (
 const GitHubFormType = ({ modals }: GitHubFormTypeProps) => {
 
   const { setCreateModalState, setRulesetModalState } = modals
+  const result = api.webhook.createWebhook.useMutation()
   
   const handleSubmit = (e: React.FormEvent) => {
     e.preventDefault()
     // create automation and webhook logic func calls here
     // show some UI if worked for failed
+    console.log(result.mutate());
```

Comment:
Is this a console log with a side effect besides printing to console? 🤔

Reply:
The log isn't needed as it was used to make sure the rpc was being hit correctly on the frontend. It shows `undefined` when logging. `result.mutate()` itself does not have a side effect with the project because the webhook only works with a public url so I would need to advance this project further as a hosted project. I don't have permissions to create an ngrok or tl tunnel in the work laptop.

Reply:
However, as I implement further, I will make it a side effect by displaying the created webhook on the users home page by getting the data from the db